pyicloud/services/findmyfriends.py

```diff
@@ -0,0 +1,84 @@
+"""Find my Friends service."""
+from __future__ import absolute_import
+import json
+
+
+class FindFriendsService(object):
+    """
+    The 'Find my Friends' iCloud service
+
+    This connects to iCloud and returns friend data including the near-realtime
+    latitude and longitude.
```

Comment:
This is not near-realtime since we have to "manually" call `refresh_data` to update locations.

Also, location status may be pending with the first request + between 2 fetches depending on the interval, so we are not sure of the realtime.
Please remove this.

Reply:
Docstring borrowed from Find My Iphone.

"This connects to iCloud and return phone data including the near-realtime latitude and longitude."

https://github.com/picklepete/pyicloud/blob/master/pyicloud/services/findmyiphone.py#L13

Reply:
You are talking about stuffs that are going to be changed : #227

Please apply the comment

Reply:
Ok. For clarity, this PR is from 2017 and the PR you mentioned isn't merged yet...

Happy to remove "real-time" from the doc-string